examples/relu/relu_metal/relu.mm

```diff
@@ -30,16 +23,17 @@
 
     int numThreads = input.numel();
 
-    // Construct the full path to the metallib file
-    std::string moduleDir = getModuleDirectory();
-    std::string metallibPath = moduleDir + "/" + METALLIB_PATH;
+    // Load the embedded Metal library from memory
+    dispatch_data_t libraryData = dispatch_data_create(
```

Comment:
Maybe we could even make this friendlier for Metal kernels by generating/providing a header that does this for you? So something like

```
id<MTLLibrary> customKernelLibrary = [ourFunction args];
```

(Not sure what the exact syntax should look like, since I'm not familiar with ObjectiveC++).

Not 100% sure if this is useful, but maybe worth experimenting with?

It's nice to see the path handling go away!